testing/describes.js

```diff
@@ -483,6 +486,7 @@ class IntegrationFixture {
     if (env.iframe.parentNode) {
       env.iframe.parentNode.removeChild(env.iframe);
     }
+    return RequestBank.tearDown();
```

Comment:
To make sure `RequestBank` is only used with integrationFixture? I don't see an issue with using it in RealWinFixture? 
However I don't know if it is too expensive to do a xhr request for every test.

Reply:
It's currently only used in IntegrationFixture. At this moment I don't see much value for unit test.

Reply:
Sounds good. do you have an answer to? I guess that's fine since it's only IntegrationFixture. 
>However I don't know if it is too expensive to do a xhr request for every test.

Reply:
The cost is a couple of `ms` per test case (which is the delay of a local XHR).